### PR TITLE
AM-6265: Update Kubeslice Controller readme file

### PR DESCRIPTION
**Pull request by uma-kt**
- Fixed hyperlinks 
- Rephrased the doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +56,7 @@ cd kubeslice-controller
 ```console
 make docker-build
 ```
-### Running Local Image on Kind Clusters
+### Running Local Image on a Kind Clusters
```
Comment (veenamj-avesha) [inline README.md:59]: on a Kind Cluster  

a is not required for plurals

File: README.md [hunk 2]
```diff
@@ -111,22 +114,23 @@ kubeslice:
    make chart-deploy VALUESFILE=yourvaluesfile.yaml
    ```
    
-### Verify if the Operator is Running
+### Verify the installation
```
Comment (veenamj-avesha) [inline README.md:117]: Installation

File: README.md [hunk 3]
```diff
@@ -12,17 +12,20 @@ using [custom resource definitions (CRDs)](https://kubernetes.io/docs/tasks/acce
...
 
 The KubeSlice Controller orchestrates the creation and management of slices on worker clusters.
-It is strongly recommended to use a released version. Follow the instructions provided in this [document](https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller).
+It is strongly recommended to use a released version. Follow the instructions provided in this [document](https://kubeslice.io/documentation/open-source/0.5.0/cloud-cluster-quick-start).
 
-## Building and Deploying `kubeslice-controller` in a Local Kind Cluster
-For more information, see [getting started with kind clusters](https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-kind-clusters).
+## Building and Deploying a KubeSlice Controller on a Local Kind Cluster
+
+See our documentation on [installing the KubeSlice Controller on a Kind Cluster](https://kubeslice.io/documentation/open-source/0.5.0/tutorials/kind-install-kubeslice-controller).
```
Comment (veenamj-avesha) [inline README.md:19]: on a kind cluster

File: README.md (hunk 1 above)
Comment (uma-kt) [reply README.md:59]: done

File: README.md (hunk 2 above)
Comment (uma-kt) [reply README.md:117]: done

File: README.md (hunk 3 above)
Comment (uma-kt) [reply README.md:19]: done

Comment (veenamj-avesha) [reply README.md:19]: on a kind cluster - not `on a Kind Cluster`

Comment (uma-kt) [reply README.md]: updated

**Pull request merged**